content/en/docs/instrumentation/java/manual.md

```diff
@@ -7,31 +7,198 @@ aliases:
   - /docs/instrumentation/java/manual_instrumentation
 weight: 30
 description: Manual instrumentation for OpenTelemetry Java
-cSpell:ignore: autoconfigure classpath customizer logback loggable multivalued
+# prettier-ignore
+cSpell:ignore: autoconfigure Autowired classpath customizer logback loggable multivalued rolldice springframework
 ---
 
 <!-- markdownlint-disable no-duplicate-heading -->
 
 {{% docs/instrumentation/manual-intro %}}
 
-## Setup
+{{% alert title="Note" color="info" %}}
 
-The first step is to get a handle to an instance of the `OpenTelemetry`
-interface.
+On this page you will learn how you can add traces, metrics and logs to your
+code _manually_. But, you are not limited to only use one kind of
+instrumentation: use
+[automatic instrumentation](/docs/instrumentation/java/automatic/) to get
```

Comment:
Not sure of exactly phrasing, but it would be nice to have something here that also points out the use case of dev/operators who don't have access to the source code or otherwise can't or don't want to modify source. Maybe something like:

"If you cannot modify the source code for your project, you should skip manual instrumentation and just use automatic instrumentation." 

Feel free to improve on that. :)

Reply:
I added a sentence, please take a look